com.unity.ml-agents/CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0.0-preview]
```

Comment:
Package verification will probably complain about this. You might need to mark the old Unreleased section as 1.9.0, and this as Unreleased